format/testdata/cond.clj

```diff
@@ -15,3 +15,59 @@
     (assoc :foo 1
            :bar
              234))
+
+(condp x y a 1 b 2 c :>> f)
```

Comment:
Minor quibble, but IMO these test cases are hard to follow because they're so concise and don't resemble real code. The `issue69.clj` test case, for instance, is easy to look at and figure out what I would expect the indentation to be. With these, I have to pretty carefully parse them to figure out what each character corresponds to.